src/Components/ArticleTemplate.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import {Container, Divider,Header, Placeholder} from 'semantic-ui-react'
+
+class ArticleTemplate extends Component {
```

Comment:
Again, why use class component here?

Reply:
See discussion above.  Since this does not have a state, it can probably be refactored into an `element`, but I would have to check this to make sure.  Perhaps the cohort can have a quick discussion about the use of `function` and `class` components, and static `elements`.

Reply:
See comment above.

Reply:
Fixed.